lib/simulator-xcode-9.js

```diff
@@ -51,8 +52,14 @@ class SimulatorXcode9 extends SimulatorXcode8 {
   }
 
   async shutdown () {
+    await restoreTouchEnrollShortcuts();
```

Comment:
Yes. As of now we have no tests, and no sense that running in parallel will work anyway. This PR is just to get the machinery in place to be able to test such things, while maintaining the current functionality.